src/state/airdrop/reducer.ts

```diff
@@ -22,6 +28,37 @@ export const airdropReducer = (state = initialState, action: AirdropActionType):
         ...state,
         thankYouFlowCompleted: true,
       };
+    case AirdropAction.CheckSubscription:
```

Comment:
use waterfall for the same reducers
case AirdropAction.SubscribeWallet:
case AirdropAction.CheckSubscription:

Reply:
done